src/vertex/app/changelog.md

```
@@ -4,6 +4,67 @@
+<details>
+<summary><strong>Feature Updates (1)</strong></summary>
+
+- **Cohort Visibility Toggle**: Moved the visibility management (Public vs Private) directly onto the **Cohort Details Card**. Users can now toggle a cohort's visibility with a switch, complete with a confirmation dialog explaining the privacy implications.
+
+</details>
+
+<details>
+<summary><strong>UI Enhancements (3)</strong></summary>
+
+- **Dark Mode Styling**:
+    - **Sidebar**: Standardized active state navigation to use a branded dark blue theme (`dark:bg-blue-900/20`, `dark:text-blue-500`) across both Primary and Secondary sidebars, ensuring high contrast and consistency.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)